From 6c2e86cb882b42a91644c30d7dd022bd3c910ada Mon Sep 17 00:00:00 2001
From: Vol Zhdanov <wight554@gmail.com>
Date: Wed, 6 Feb 2019 22:04:06 +0000
Subject: [PATCH 3/4] qcom: label persist files with /(mnt/vendor)/persist
 instead of /mnt/vendor/persist

* not to force everyone to move persist folder
* not to force everyone who can't/don't want to move persist to use legacy

Change-Id: Iaae2d01d6733e59f5158e63697fdec681318401e
---
 vendor/common/file_contexts | 40 ++++++++++++++++++-------------------
 1 file changed, 20 insertions(+), 20 deletions(-)

diff --git a/vendor/common/file_contexts b/vendor/common/file_contexts
index 98a4ce6f..8241c68c 100644
--- a/vendor/common/file_contexts
+++ b/vendor/common/file_contexts
@@ -594,26 +594,26 @@
 # persist files
 #
 /persist(/.*)?                                                                 u:object_r:mnt_vendor_file:s0
-/mnt/vendor/persist/bluetooth(/.*)?                                            u:object_r:persist_bluetooth_file:s0
-/mnt/vendor/persist/drm(/.*)?                                                  u:object_r:persist_drm_file:s0
-/mnt/vendor/persist/sensors(/.*)?                                              u:object_r:sensors_persist_file:s0
-/mnt/vendor/persist/alarm(/.*)?                                                u:object_r:persist_alarm_file:s0
-/mnt/vendor/persist/time(/.*)?                                                 u:object_r:persist_time_file:s0
-/mnt/vendor/persist/data(/.*)?                                                 u:object_r:persist_drm_file:s0
-/mnt/vendor/persist/data/tz(/.*)?                                              u:object_r:persist_drm_file:s0
-/mnt/vendor/persist/data/sfs(/.*)?                                             u:object_r:persist_drm_file:s0
-/mnt/vendor/persist/qti_fp(/.*)?                                               u:object_r:persist_qti_fp_file:s0
-/mnt/vendor/persist/usf(/.*)?                                                  u:object_r:persist_usf_file:s0
-/mnt/vendor/persist/hlos_rfs(/.*)?                                             u:object_r:persist_rfs_shared_hlos_file:s0
-/mnt/vendor/persist/display(/.*)?                                              u:object_r:persist_display_file:s0
-/mnt/vendor/persist/rfs.*                                                      u:object_r:persist_rfs_file:s0
-/mnt/vendor/persist/speccfg(/.*)?                                              u:object_r:regionalization_file:s0
-/mnt/vendor/persist/misc(/.*)?                                                 u:object_r:persist_misc_file:s0
-/mnt/vendor/persist/bms(/.*)?                                                  u:object_r:persist_bms_file:s0
-/mnt/vendor/persist/vpp(/.*)?                                                  u:object_r:persist_vpp_file:s0
-/mnt/vendor/persist/secnvm(/.*)?                                               u:object_r:persist_secnvm_file:s0
-/mnt/vendor/persist/FTM_AP(/.*)?                                               u:object_r:vendor_persist_mmi_file:s0
-/mnt/vendor/persist/hvdcp_opti(/.*)?                                           u:object_r:persist_hvdcp_file:s0
+/(mnt/vendor)/persist/bluetooth(/.*)?                                            u:object_r:persist_bluetooth_file:s0
+/(mnt/vendor)/persist/drm(/.*)?                                                  u:object_r:persist_drm_file:s0
+/(mnt/vendor)/persist/sensors(/.*)?                                              u:object_r:sensors_persist_file:s0
+/(mnt/vendor)/persist/alarm(/.*)?                                                u:object_r:persist_alarm_file:s0
+/(mnt/vendor)/persist/time(/.*)?                                                 u:object_r:persist_time_file:s0
+/(mnt/vendor)/persist/data(/.*)?                                                 u:object_r:persist_drm_file:s0
+/(mnt/vendor)/persist/data/tz(/.*)?                                              u:object_r:persist_drm_file:s0
+/(mnt/vendor)/persist/data/sfs(/.*)?                                             u:object_r:persist_drm_file:s0
+/(mnt/vendor)/persist/qti_fp(/.*)?                                               u:object_r:persist_qti_fp_file:s0
+/(mnt/vendor)/persist/usf(/.*)?                                                  u:object_r:persist_usf_file:s0
+/(mnt/vendor)/persist/hlos_rfs(/.*)?                                             u:object_r:persist_rfs_shared_hlos_file:s0
+/(mnt/vendor)/persist/display(/.*)?                                              u:object_r:persist_display_file:s0
+/(mnt/vendor)/persist/rfs.*                                                      u:object_r:persist_rfs_file:s0
+/(mnt/vendor)/persist/speccfg(/.*)?                                              u:object_r:regionalization_file:s0
+/(mnt/vendor)/persist/misc(/.*)?                                                 u:object_r:persist_misc_file:s0
+/(mnt/vendor)/persist/bms(/.*)?                                                  u:object_r:persist_bms_file:s0
+/(mnt/vendor)/persist/vpp(/.*)?                                                  u:object_r:persist_vpp_file:s0
+/(mnt/vendor)/persist/secnvm(/.*)?                                               u:object_r:persist_secnvm_file:s0
+/(mnt/vendor)/persist/FTM_AP(/.*)?                                               u:object_r:vendor_persist_mmi_file:s0
+/(mnt/vendor)/persist/hvdcp_opti(/.*)?                                           u:object_r:persist_hvdcp_file:s0
 ###################################
 # persist changes for backword comptaibily
 # this changes would be removed once tech team make the changes
-- 
2.17.1

